grails-app/views/linkWizard/index.gsp

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Is there a reason we have to have this as a separate react app outside of where these questions lie now? I worry about us propagating this anti-pattern unless absolutely necessary. If possible, reference the `LinkWizard` component in the current project wizard. 

Caveat - I have not looked at this in the UI so I don't know if the above advice is possible or not. What I do know is that we are increasing the technical debt of [BTRX-609](https://broadinstitute.atlassian.net/browse/BTRX-609) every time we do this. It would be best to make an effort to integrate this now if possible.

Reply:
I think this will change when we implement all the application in a react-way. As I understand this should live as a separate app meanwhile.

Reply:
To get a better handle on this, I spun up this branch and looked around. 

1. There is a link to this wizard from the list of consent groups in a project
2. The consent group tab is populated with an ajax call to the controller via jquery: `$("#consent-groups").load(...`
3. The controller renders a gsp which is not converted yet. 

The specific story this PR will impact is: [BTRX-613](https://broadinstitute.atlassian.net/browse/BTRX-613)
The net effect is that we're increasing the scope of that ticket (or any other ticket) when we add new functionality this way.

Reply:
I totally agree with you about this caveat.
Is there another way to implement this view in the meantime?

Reply:
I think what you have here @tadeork is the best interim solution. I was hoping that the wizard component could be called from another, pre-existing react component that was already written. I had to run it locally to realize that would not work.